data/hai.txt

```diff
@@ -465,19 +465,27 @@ phrase "friendly unfettered"
 	word
 		"We "
```

Comment:
Still have to deal with this. https://github.com/endless-sky/endless-sky/blob/master/data/hails.txt#L1387